src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,74 @@
+package duke;
+
+import duke.dependencies.parser.Controller;
+
+
+/**
+ * Class that separates the command checker/parser from the main(). Catches command that
+ * ends/closes program, terminating it immediately. Prints to std::out.
+ *
+ */
+class Ui {
+    private static final String DIVIDER = "____________________________________________________________\n";
+    private static final String GREETING = "Hello, I'm Duke\nwhat can I do for you?\n";
+    private static final String LOGO = " ____        _        \n"
+                                    + "|  _ \\ _   _| | _____ \n"
+                                    + "| | | | | | | |/ / _ \\\n"
+                                    + "| |_| | |_| |   <  __/\n"
+                                    + "|____/ \\__,_|_|\\_\\___|\n";
+    private static final String CIAO = DIVIDER + "Spero di rivederti presto\n" + DIVIDER;
+    private static final String CONVO_START = DIVIDER + LOGO + "\n" + GREETING + DIVIDER;
+    private static final String END = "end|ciao|bye|close|exit|nights|shutdown";
+
+    private  static final Controller CONTROLLER = Controller.initController();
+
+
+    /**
+     * Starts the Duke application.
+     */
+    public void start() {
+        System.out.println(CONVO_START);
+    }
+
+    public boolean checkAuth() {
```

Comment:
Can consider writing java docs for these methods (even though its trivial) because based on the coding standards all public class and methods should have headers.